template/page-studs/_app/with-trpc.tsx

```diff
@@ -13,7 +13,7 @@ const getBaseUrl = () => {
   if (typeof window !== "undefined") {
     return "";
   }
```

Comment:
here ^

Reply:
idk how I missed this .. sry .. but then isn't the next process.browser technically the same thing shouldn't u  remove it?? or  it's there just to be sure